pom.xml

```diff
@@ -2120,6 +2120,23 @@
                 </rules>
               </configuration>
             </execution>
+            <execution>
+              <id>ban-jetty-stringutil</id>
+              <goals>
+                <goal>enforce</goal>
+              </goals>
+              <phase>process-sources</phase>
+              <configuration>
+                <rules>
+                  <restrictImports>
+                    <reason>Jetty's StringUtil is disabled by default.  Replace it with Apache Common's StringUtils.</reason>
+                    <bannedImports>
+                      <bannedImport>org.eclipse.jetty.util.StringUtil</bannedImport>
+                    </bannedImports>
+                  </restrictImports>
```

Comment:
No need for a new `<execution>`, please move this to existing one:

https://github.com/apache/ozone/blob/25ba1be825814e63766ab99132342b46b282ad9b/pom.xml#L1977-L1985